litrpc/lit-autopilot.yaml

```diff
@@ -0,0 +1,17 @@
+type: google.api.Service
+config_version: 3
+
+http:
+  rules:
+
+    # lit-autopilot.proto
+    - selector: litrpc.Autopilot.ListAutopilotFeatures
```

Comment:
Same here, could be a bit more aligned to common REST patterns (which we don't use consistently everywhere, I know)...

Reply:
what would you recommend for the "revoke" one? since it is not really a delete but rather an update...

Reply:
should it be like this? or should it just be a delete? and is the "local_public_key" part ok?

```
post: "/v1/autopilot/sessions/{local_public_key}"
```

Reply:
note that the local_public_key is bytes.... so would require the base64url format you mentioned right?

Reply:
I think `delete` still makes sense here, even if it doesn't fully remove the session. The public key being `bytes` does mean the base64url format is required. We solve this in `lnd` by adding a specific comment to the field itself, like something like `When using REST, this field must be encoded as base64url.` which will then show up in the API docs.

Reply:
cool - updated to add that comment to the RevokeSession and RevokeAutopilotSession methods 👍  thanks for the insights!